### PR TITLE
Add JSON API version of Cuenote Import Workflow

### PR DESCRIPTION
**Pull request by hjmsano**
Cuenote has two API version, one is XML SOAP, another one is REST JSON. (depends on the contract between end-user and YMIRLINK)
This WF is for REST JSON API, but basically the behavior is the same with XML version.

### CODE REVIEW

**Comment by chezou** [comment]
Didn't look at a diff between JSON version and XML one yet, should we make a separate directory? Personally, I prefer to have one directory for the same purpose workflows, and it would be less confusing for customers as well, I think.

**Comment by hjmsano** [comment]
I was thinking to do that but there is a few difference, not just a response type.
- JSON version does not require queueing.
- Slightly different columns.
- A process for building a request... XML version uses POST&body but JSON version uses GET with structured path.

On Cuenote side, both APIs are independent and it looks maintained separately.

File: integration-box/cuenote-import-json/get_jobs.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import cuenote
+import json
+import pandas
+import time
+import os
+import sys
+from datetime import datetime
+
+os.system(f"{sys.executable} -m pip install -U pytd==1.0.0 td-client")
```
Comment (chezou) [inline integration-box/cuenote-import-json/get_jobs.py:9]: The latest version of pytd is 1.2.0.

```suggestion
os.system(f"{sys.executable} -m pip install -U pytd==1.2.0 td-client")
```

File: integration-box/cuenote-import-json/get_logs.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import cuenote
+import pandas
+import io
+import os
+import sys
+
+os.system(f"{sys.executable} -m pip install -U pytd==1.0.0 td-client")
```
Comment (chezou) [inline integration-box/cuenote-import-json/get_logs.py:7]:
```suggestion
os.system(f"{sys.executable} -m pip install -U pytd==1.2.0 td-client")
```

**Comment by chezou** [comment]
hmm, in [recommendation box](https://github.com/treasure-data/treasure-boxes/tree/master/machine-learning-box/recommendation), there are three different types of implementation as a separate workflow/code. I thought not to integrate into one code, but make in one directory still could be reasonable.

File: integration-box/cuenote-import-json/get_jobs.py (hunk 1 above)
Comment (hjmsano) [reply integration-box/cuenote-import-json/get_jobs.py]: updated

File: integration-box/cuenote-import-json/get_logs.py (hunk 2 above)
Comment (hjmsano) [reply integration-box/cuenote-import-json/get_logs.py]: updated

**Comment by hjmsano** [comment]
- Updated `pytd` version to `1.2.0`
- Merged two directories into `cuenote-integration`

**Pull request merged**